symplyphysics/laws/optics/radius_newton_ring_from_wavelength_order_radius_curvature_refractive_index.py

```diff
@@ -0,0 +1,55 @@
+from sympy import (Eq, solve, sqrt)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless,
+    angle_type
+)
+
+# Description
+## Newton's rings are annular interference maxima and minima that appear around the point of contact between
+## a convex lens and a plane—parallel plate when light passes through the lens and plate.
+## The order of interference indicates the number of whole wavelengths. When the difference in the path of the
+## beam reflected from the convex surface of the lens at the glass-air boundary and the beam reflected from the
+## plate at the air-glass boundary is equal to an entire wave "n", then these two waves hit the observation
+## point with the same faces and demonstrate interference.
+
+## Law is: r = sqrt(k * R * L / n), where
```

Comment:
Could you show me the source of this formula?

Reply:
> Could you show me the source of this formula?

https://ru.wikipedia.org/wiki/Кольца_Ньютона

If instead of k we substitute k = 1, 2, 3, ..., then this is the minimum condition(dark rings). If instead of k we substitute k = 1/2, 3/2, 5/2, ..., then this is the maximum condition(light rings).

I can add this to the description. Is it okay?

Reply:
Yes, you can rephrase this law, that it describes dark Newton's rings.
But you cannot make it describe light rings because you have integer in `calculate` function as parameter.

Reply:
What if user sends `0.3` as order of interference? Does it have any sense for this law?
If not, you should prevent it from happening.